x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardFollowTasksExecutor.java

```diff
@@ -364,7 +364,7 @@ protected void innerUpdateAliases(final LongConsumer handler, final Consumer<Exc
                     if (aliasActions.isEmpty()) {
                         handler.accept(leaderIndexMetadata.getAliasesVersion());
                     } else {
-                        final var request = new IndicesAliasesRequest().masterNodeTimeout(TimeValue.MAX_VALUE);
+                        final var request = new IndicesAliasesRequest(TimeValue.MAX_VALUE, TimeValue.ZERO);
```

Comment:
Is this `TimeValue.ZERO` intentional here? If so, what was your reasoning?

Reply:
Yes this is a slight change. We don't care about acking here, no need to wait to see if this cluster state update is applied everywhere.